src/index.ts

```diff
@@ -1,5 +1,4 @@
 import { TinyEmitter } from 'tiny-emitter';
-import { v4 as uuidv4 } from 'uuid';
```

Comment:
do we use it for something else? can we remove this from package dependencies?

Reply:
Good point. I haven't checked, but I'm pretty sure I wasn't the one to add it. Let's see.

Reply:
We use it in `src/events-handler.ts` too. We could switch that one for the same impl potentially, but I don't really wanna touch that now if we don't have to?